mne/forward/_field_interpolation.py

```diff
@@ -125,9 +134,9 @@ def _map_meg_channels(inst, pick_from, pick_to, mode='fast'):
     mapping : array
         A mapping matrix of shape len(pick_to) x len(pick_from).
     """
-    info_from = pick_info(inst.info, pick_from, copy=True)
-    info_to = pick_info(inst.info, pick_to, copy=True)
-
+    if picks is not None:
+        info_from = pick_info(info_from, picks)
+        info_to = pick_info(info_to, picks)
     # no need to apply trans because both from and to coils are in device
```

Comment:
@mainakjas, could you clarify? this comment isn't necessarily true you are supplying the `dev_head_t` that is used in `_create_meg_coils` to transform them to head space.